Dockerfile

```diff
@@ -69,8 +69,13 @@ RUN \
     /tmp/* \
     /var/tmp/*
 
-# expose the working directory, the Tomcat port, the BrowserSync ports
 USER jhipster
+
+RUN \
```

Comment:
This code: `RUN mkdir /home/jhipster/app` ~will~ add another layer for the final Docker image
You can compare: `docker image inspect <old image>` with `docker image inspect <new image>`